### PR TITLE
Enhance dockerfile

### PR DESCRIPTION
**Pull request by krlsdu**
When you create the image the command "mix deps.get " include the dependencies in a separate layer, so, the second command, "mix meditate" launch this error:

> Unchecked dependencies for environment dev:
> _* file_system (Hex package)
>   the dependency is not available, run "mix deps.get"
> ** (Mix) Can't continue due to errors on dependencies

Because not is the same session and that layer created not is accessible
The improve resolve that problem.

The last improve is the alpine image, because of the size.
In my experiments, the size of resulted images is big different.
Not alpine base image, the size of the resulted image is 1.1GB
Using alpine base image the size of the resulted image is 90.3MB

### CODE REVIEW

**Comment by krlsdu** [comment]
Ok @iamvery.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Elixir Koans
 
-[![Build Status](https://travis-ci.org/elixirkoans/elixir-koans.svg?branch=master)](https://travis-ci.org/elixirkoans/elixir-koans)
+[![Build Status](https://travis-ci.org/krlsdu/elixir-koans.svg?branch=master)](https://travis-ci.org/krlsdu/elixir-koans)
```
Comment (felipesere) [inline README.md]: This and the other badges seem like a leftover. 
We might be interested to pull some of those things over to the elixirkoans organisation though 🤔

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (felipesere) [inline .travis.yml]: What does this “+” do?

**Comment by felipesere** [comment]
I am not 100% sure I understand the issue with running mix deps.get in a separate layer.
My expectation was that all the changes in the file system would be reflected in later layers?
It could be tha my understanding here is wrong though 😊

I’ll give this PR a try later when I get home

File: .travis.yml (hunk 2 above)
Comment (krlsdu) [reply .travis.yml]: My mistake fix o commit 
4adfafb

File: README.md (hunk 1 above)
Comment (krlsdu) [reply README.md]: My mistake fix on commit 
06187ba

**Pull request closed without merging**